src/quickgui/qgsquickcoordinatetransformer.cpp

```diff
@@ -76,7 +76,14 @@ void QgsQuickCoordinateTransformer::setSourceCrs( const QgsCoordinateReferenceSy
 
 void QgsQuickCoordinateTransformer::setTransformContext( const QgsCoordinateTransformContext &context )
 {
-  mCoordinateTransform.setContext( context );
+  mTransformContext = context;
```

Comment:
In most cases the property setters have a `if ( mCoordinateTransform.context() == context )` guard in place to avoid binding loops. Would this work with the transform context?

Reply:
Not currently - there's no equality operator for transform contexts, so that would need to be added first.